src/amo/components/SectionLinks/styles.scss

```diff
@@ -15,6 +15,7 @@
 .SectionLinks-link:link {
   color: $grey-40;
   display: inline-block;
+  font-size: $font-size-m;
```

Comment:
what has this changed from? (it'll affect the entire frontend site, right?)

Reply:
This is the same font-size as before, i.e. the `body` font size. It's needed here because the style that is bundled in the `blog-utils` library does not contain the `body` font size. It only contains the CSS for the components that are rendered.